podcasts/Sharing/SharingModal.swift

```diff
@@ -162,6 +162,24 @@ extension SharingModal.Option {
         return imageInfo
     }
 
+    func itemProvider(style: ShareImageStyle) -> NSItemProvider {
+        switch self {
+        case .clipShare(_, _, _, let progress):
+            if let fileURL = progress.fileURL {
```

Comment:
Maybe use a `guard `here instead of an  `if`

Reply:
The only issue is I have to add `return` in each block:

Personally that doesn't bother me too much, though.

```suggestion
    func itemProvider(style: ShareImageStyle) -> NSItemProvider {
        switch self {
        case .clipShare(_, _, _, let progress):
            guard let fileURL = progress.fileURL else {
                return ShareImageView(info: imageInfo, style: style, angle: .constant(0)).itemProvider()
            }
            if #available(iOS 16.0, *) {
                return NSItemProvider(contentsOf: fileURL, contentType: UTType(filenameExtension: fileURL.pathExtension) ?? .mpeg4Movie)
            } else {
                return NSItemProvider(contentsOf: fileURL)!
            }
        default:
            return ShareImageView(info: imageInfo, style: style, angle: .constant(0)).itemProvider()
        }
    }
```